server/eclipse-project/src/main/java/au/org/democracydevelopers/corla/endpoint/EstimateSampleSizes.java

```diff
@@ -1,73 +1,258 @@
 package au.org.democracydevelopers.corla.endpoint;
 
+import au.org.democracydevelopers.corla.model.ContestType;
+import au.org.democracydevelopers.corla.model.IRVComparisonAudit;
 import spark.Request;
 import spark.Response;
 import us.freeandfair.corla.asm.ASMEvent;
+import us.freeandfair.corla.controller.ContestCounter;
 import us.freeandfair.corla.endpoint.AbstractDoSDashboardEndpoint;
 
 import org.apache.log4j.LogManager;
 import org.apache.log4j.Logger;
+import us.freeandfair.corla.model.*;
+import us.freeandfair.corla.persistence.Persistence;
+import us.freeandfair.corla.query.CountyContestResultQueries;
+
+import java.math.BigDecimal;
+import java.math.MathContext;
+import java.text.DecimalFormat;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static org.apache.commons.text.StringEscapeUtils.escapeCsv;
+import static us.freeandfair.corla.controller.ComparisonAuditController.createAuditOfCorrectType;
 
```

Comment:
If we say that is can be used before or during then we should test it being used during.

Reply:
Actually given today's meeting UI discussion, I'll just remove the claim that it can be used during the audit, since nobody seems to want that anyway. I guess we should test that it doesn't break anything (else) if accidentally called during the audit, but I don't see how it could since it doesn't alter the database.